projects/aca-folder-rules/src/lib/model/rule.model.ts

```diff
@@ -34,7 +35,7 @@ export interface Rule {
   asynchronous: boolean;
   errorScript: string;
   shared: boolean;
-  triggers: ('inbound' | 'update' | 'outbound')[];
+  triggers: ('INBOUND' | 'UPDATE' | 'OUTBOUND')[];
```

Comment:
I think these should be lower case. If the API is returning upper case then it's a bug.

Reply:
> I think these should be lower case. If the API is returning upper case then it's a bug.

Yes, you are right. It was a bug, but now it is fixed as far as I can see.